qt/python/mantidqt/mantidqt/widgets/helpwindow/helpwindowmodel.py

```diff
@@ -5,10 +5,29 @@
 #   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 # SPDX - License - Identifier: GPL - 3.0 +
 import os
+
+from mantid import logger
 from qtpy.QtCore import QUrl
 from qtpy.QtWebEngineCore import QWebEngineUrlRequestInterceptor, QWebEngineUrlRequestInfo
 
 
+def getMantidVersionString():
```

Comment:
Functions are easier to read when there is only one `return`. I suggest creating an object with an appropriate "default" value. In this case `versionstr="nightly"` would probably get to something useful when constructing the url later.